src/Estimators/tests/RobotDynamicsEstimator/RobotDynamicsEstimationTest.cpp

```diff
@@ -293,9 +426,13 @@ TEST_CASE("RobotDynamicsEstimator Test")
         REQUIRE((output.tau_F - dataset.expectedTauF.row(sample).transpose()).isZero(0.1));
         REQUIRE((output.tau_m - dataset.expectedTaum.row(sample).transpose()).isZero(0.1));
         REQUIRE(((output.tau_m - output.tau_F) - dataset.expectedTauj.row(sample).transpose()).isZero(0.1));
-        for (auto const& [key, value] : dataset.fts)
+        for (int idx = 0; idx < sensors["ft"].size(); idx++)
+        {
+            REQUIRE((output.ftWrenches[sensors["ft"][idx].ukfSensorName] - dataset.fts[sensors["ft"][idx].sensorName].row(sample).transpose()).isZero(0.1));
```

Comment:
You can use `isApprox` (https://eigen.tuxfamily.org/dox/classEigen_1_1DenseBase.html#ae8443357b808cd393be1b51974213f9c)

```suggestion
            REQUIRE((output.ftWrenches[sensors["ft"][idx].ukfSensorName].isApprox( dataset.fts[sensors["ft"][idx].sensorName].row(sample).transpose()));
```